### PR TITLE
Add Cache Support for Next@14.1.0

### PR DESCRIPTION
**Pull request by Manuel-Antunes**
## Related Issue
https://github.com/sst/open-next/issues/354

## Proposed Changes
- create `utils` file overriding the `cacheHander` property

## Changelog
- [feat: add cache support for next@14.1.0](https://github.com/sst/open-next/commit/fa57895777101744ad03b2a05b3f2ec972aceb86)
- [fix: lint files](https://github.com/sst/open-next/commit/671dd86dc77e5f908bee4cdeb2d682beb00d7216)

### CODE REVIEW

**Review by conico974 — CHANGES_REQUESTED** [review]
I've added some comment, if you could change that.

File: packages/open-next/src/adapters/plugins/14.1/serverHandler.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/*eslint-disable simple-import-sort/imports */
```
[the remark this answers is not in the document]
Comment (conico974) [reply packages/open-next/src/adapters/plugins/14.1/serverHandler.ts]: That's not how the plugin works https://open-next.js.org/inner_workings/plugin
Long story short, the only things that happens is that everything between `//#override` and `//#endOverride` is replaced in the files you provide

**Review by conico974 — APPROVED** [review]
LGTM, i'll probably merge this by the end of the day

**Comment by Manuel-Antunes** [comment]
Awesome!

**Pull request merged**